cmd/api-errors.go

```diff
@@ -698,6 +743,23 @@ func toAPIErrorCode(err error) (apiErr APIErrorCode) {
 		return apiErr
 	}
 
+	switch err { // SSE errors
+	case errInsecureSSERequest:
```

Comment:
Perhaps add test cases for these entries for their proper translation to right errorCode.

Reply:
done